cmd/scaffold.go

```
@@ -34,11 +37,40 @@ func ScaffoldInit(ctx *cli.Context) error {
 	}
 	err = sc.Organize()
 	if err != nil {
-		return err
+		// If we encounter an error setting up the project files, see if the user
+		// wants to overwrite them; if not, return w/o error to end running the
+		// scaffold init command.
+		if promptForOverwrite() {
+			forceFn := scaffold.Force(true)
+			forceFn(sc)
+			// If an error happens with --force enabled, something new has gone wrong.
+			if err = sc.Organize(); err != nil {
+				return err
+			}
+		}
+		return nil
 	}
 	err = sc.Init()
 	if err != nil {
 		return err
 	}
 	return nil
 }
+
+func promptForOverwrite() bool {
+	prompt := promptui.Prompt{
+		Label:   "Scaffold already initialed; do you want to overwrite (y/N)?",
+		Default: "N",
+		Validate: func(input string) error {
+			input = strings.ToLower(input)
+			if input != "y" && input != "n" {
+				return errors.New("invalid response: must be 'y' or 'n'")
+			}
+			return nil
+		},
+	} // Not using IsConfirm; it seems to have rendering issues in a few terminal cases.
+	if result, err := prompt.Run(); err == nil && result == "y" {
```

Should we propagate the error if `err != nil` ?

---

We can; I had thought that in the event where Prompt.Run failed something weird enough happened that we should just do nothing (e.g. return `nil`) but that does obfuscate that something has indeed gone wrong.

I'll update to return the error; when expanding the use of promptui maybe I can put together something better to handle `Run` errors than just returning it to the the user.